examples/oft-solana/tasks/solana/index.ts

```diff
@@ -155,22 +157,67 @@ export const getAddressLookupTable = async (connection: Connection, umi: Umi, fr
     }
 }
 
+export enum TransactionType {
+    CreateToken = 'CreateToken',
+    CreateMultisig = 'CreateMultisig',
+    InitOft = 'InitOft',
+    SetAuthority = 'SetAuthority',
+    InitConfig = 'InitConfig',
+    SendOFT = 'SendOFT',
+}
+
+const TransactionCuEstimates: Record<TransactionType, number> = {
+    // for the sample values, they are: devnet, mainnet
+    [TransactionType.CreateToken]: 125_000, // actual sample: (59073, 123539), 55785 (more volatile as it has CPI to Metaplex)
+    [TransactionType.CreateMultisig]: 5_000, // actual sample: 3,230
+    [TransactionType.InitOft]: 70_000, // actual sample: 59207, 65198 (note: this is the only transaction that createOFTAdapter does)
+    [TransactionType.SetAuthority]: 8_000, // actual sample: 6424, 6472
+    [TransactionType.InitConfig]: 42_000, // actual sample: 33157, 40657
+    [TransactionType.SendOFT]: 230_000, // actual sample: 217,784
+}
+
 export const getComputeUnitPriceAndLimit = async (
     connection: Connection,
     ixs: Instruction[],
     wallet: KeypairSigner,
-    lookupTableAccount: AddressLookupTableAccount
+    lookupTableAccount: AddressLookupTableAccount,
+    transactionType: TransactionType
 ) => {
     const { averageFeeExcludingZeros } = await getFee(connection)
     const priorityFee = Math.round(averageFeeExcludingZeros)
     const computeUnitPrice = BigInt(priorityFee)
 
-    const computeUnits = await getSimulationComputeUnits(
-        connection,
-        ixs.map((ix) => toWeb3JsInstruction(ix)),
-        toWeb3JsPublicKey(wallet.publicKey),
-        [lookupTableAccount]
-    )
+    let computeUnits
+
+    try {
+        computeUnits = await backOff(
+            () =>
+                getSimulationComputeUnits(
+                    connection,
+                    ixs.map((ix) => toWeb3JsInstruction(ix)),
+                    toWeb3JsPublicKey(wallet.publicKey),
+                    [lookupTableAccount]
+                ),
+            {
+                maxDelay: 3000,
```

Comment:
`3000ms` may be a bit too aggressive for maxDelay, but we can give it a go and adjust if needed.

Reply:
resolving to merge this, just wanted to ensure it was tracked somewhere